src/mpc-hc/MainFrm.h

```diff
@@ -387,11 +387,19 @@ class CMainFrame : public CFrameWnd, public CDropClient
     void StartWebServer(int nPort);
     void StopWebServer();
 
-    int GetPlaybackMode() const { return m_iPlaybackMode; }
-    bool IsPlaybackCaptureMode() const { return GetPlaybackMode() == PM_ANALOG_CAPTURE || GetPlaybackMode() == PM_DIGITAL_CAPTURE; }
+    int GetPlaybackMode() const {

```

Comment:
These changes seem unrelated. Please revert the unrelated changes and I'll have a look at astyle.